packages/supplementary-contracts/script/DeployTokenUnlocking.s.sol

```diff
@@ -23,10 +22,7 @@ contract DeployTokenUnlocking is Script {
         vm.startBroadcast();
         tokenUnlocking = deployProxy({
             impl: address(new TokenUnlocking()),
-            data: abi.encodeCall(
-                TokenUnlocking.init,
-                (CONTRACT_OWNER, TAIKO_TOKEN, SHARED_TOKEN_VAULT, GRANTEE, uint64(TGE))
-                )
+            data: abi.encodeCall(TokenUnlocking.init, (OWNER, TAIKO_TOKEN, RECIPIENT, uint64(TGE)))
```

Comment:
@adaki2004 could you help me modifiy this script to deploy multiple instance of this contract for a given list of addresses?

Reply:
Then add another script to batch vest tokens to a list of contracts with their corresponding amount of tokens.